media/css/legal/builders-challenge.scss

```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.builders-challenge-terms {
+    ol > li::marker {
+        font-weight: bold;
+    }
+
+    .nested-lists {
+        counter-reset: count;
+
+        .nested-counter-list {
+            li {
+                counter-increment: count 1;
+
+                &::marker {
+                    content: counter(count, decimal) '. ';
+                    font-weight: 500;
+                }
+            }
+        }
+
+        .deep-nested-counter-list {
```

Comment:
**suggestion**: If we need to keep the numerical nested markers for non-IE browsers, we can simplify the counter use by setting `.nested-counter-list` on the `li` containing these lists and letting the count apply to all nested `li` elements

```
.nested-counter-list {
    counter-reset: count;
    li {
        counter-increment: count 1;

        &::marker {
            content: counter(count, decimal) '. ';
            font-weight: 500;
        }
    }
}
```

Reply:
Going to see what legal says about making the nested lists alphanumeric before making a change here

Reply:
So turns out I'm going to have to keep this code even with alphanumeric values, because the nested list values need to increment after the `p` tag following the parent list.

Also, in your suggestion, it's still not taking into account the second set of the nested links :/ but maybe I misunderstood?

Since timing is a constraint, I'm going to go ahead and keep this change, but I've updated the code to show alphanumeric characters instead of numbers, for a better fallback for IE.